index.html

```diff
@@ -2,13 +2,688 @@
+            Saint Petersburg, Russia
+          </div>
+          <div class="block-three__img block-three__img--appartment">
+            <img class="block-three__img" src="././images/Rectangle%2049.png" alt="Appartment">
+            <div class="circle phone">
+              <svg
+                class="arrow phone"
+                width="9"
+                height="21"
+                viewBox="0 0 9 21"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M.941 19.823 7.1 11.612a2 2 0 0 0 0-2.4L.941 1"
+                  stroke="#383838"
+                  stroke-width="2"
+                />
+              </svg>
+            </div>
+          </div>
+          <div class="block-three__caption block-three__caption--appartment">
+            Apartment Sunshine
+          </div>
+          <div class="block-three__location block-three__location--appartment">
+            Santa Cruz de Tenerife, Spain
+          </div>
+          <div class="block-three__img block-three__img--villa desktop">
+            <img src="././images/Rectangle%2050.png" alt="Villa">
+          </div>
+          <div class="block-three__caption block-three__caption--villa desktop">
+            Villa Kunerad
+          </div>
+          <div
+            class="block-three__location block-three__location--villa desktop"
+          >
+            Vysokie Tatry, Slowakia
+          </div>
+          <div class="block-three__img block-three__img--hostel desktop">
+            <img src="././images/Rectangle%2051.png" alt="Hostel">
+            <div class="circle">
+              <svg
+                class="arrow"
+                width="9"
+                height="21"
+                viewBox="0 0 9 21"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M.941 19.823 7.1 11.612a2 2 0 0 0 0-2.4L.941 1"
+                  stroke="#383838"
+                  stroke-width="2"
+                />
+              </svg>
+            </div>
+          </div>
+          <div
+            class="block-three__caption block-three__caption--hostel desktop"
+          >
+            Hostel Friendship
+          </div>
+          <div
+            class="block-three__location block-three__location--hostel desktop"
+          >
+            Berlin, Germany
+          </div>
+        </div>
+      </div>
+    </div>
+  <!--Block 4-->
+  <div class="block-four container">
+    <header class="block-four__header center-lg">The best destinations</header>
+    <section class="block-four__navigation row">
+      <p class="block-four__navigation--p ">Regions</p>
+      <p class="block-four__navigation--p">Cities</p>
+      <p class="block-four__navigation--p desktop">Places of interest</p>
+      <p class="block-four__navigation--p desktop phone">Places</p>
+    </section>
```

Comment:
You missed header in every of sections, but actually they are not sections. Read about it [on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).

Reply:
ok.